### ISSUE TITLE
Do not log any messages prior to capturing logcat

### ISSUE THREAD

**Issue by pfn** [ACRA/acra#356]
Often times, some number of logcat lines is taken by `ACRA` when capturing crash reports, I would not want to have these lines count against my 100-200 line captures. Please report all ACRA logcat messages after the capture has been taken.

**Comment by pfn** [comment]
example
```
01-30 09:44:44.501 D/ACRA    (22538): Using default Report Fields
01-30 09:44:44.642 I/ACRA    (22538): READ_LOGS granted! ACRA can include LogCat and DropBox data.
01-30 09:44:44.697 D/ACRA    (22538): Retrieving logcat output...
```

**Comment by william-ferguson-au** [comment]
Resolved in #357
